Worker.Extension.Sql/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.Sql", "0.1.*-preview")]
```

Comment:
I don't think we need to include the minor version here too - just `0.*-preview` should be enough (assuming we plan to keep following proper semver versioning)

@chlafreniere @lucyzhang929 thoughts?

Reply:
I don't have a strong opinion here - it's pretty unlikely that we'll go to 0.2.*, but totally fine just to be a bit more inclusive version range-wise in case I'm wrong 😄

Reply:
Will update it to 0.*-preview.